msal/src/Microsoft.Identity.Client/PublicClientApplication.cs

```diff
@@ -509,6 +509,35 @@ internal IWebUI CreateWebAuthenticationDialog(UIParent parent, UIBehavior behavi
         // endif for !NET_CORE
 #endif
 
+#if MSAL_FEATURE_USERNAMEPASSWORD
+        /// <summary>
+        /// Non-interactive request to acquire a security token from the authority, via Username/Password Authentication.
+        /// Available only on .net desktop and .net core. See https://aka.ms/msal-net-up for details.

```

Comment:
Not available on Xamarin.iOS and Xamarin.Android. See https://aka.ms/msal-net-up for details 

We'll have to update https://aka.ms/msal-net-up BTW